mm2src/lp_swap/maker_swap.rs

```diff
@@ -459,7 +459,7 @@ impl MakerSwap {
     }
 
     async fn wait_taker_fee(&self) -> Result<(Option<MakerSwapCommand>, Vec<MakerSwapEvent>), String> {
-        const TAKER_FEE_RECV_TIMEOUT: u64 = 180;
+        const TAKER_FEE_RECV_TIMEOUT: u64 = 600;
```

Comment:
I guess, you increased the timeout because `TakerSwap` doesn't send the fee transaction hex until `send_outputs` waits for confirmations. What if we just don't wait for confirmations within `send_outputs`?

Reply:
Yes, I provided more info on this in https://github.com/KomodoPlatform/atomicDEX-API/pull/1067#discussion_r717562430. I should also add that I don't like it for now, but I keep thinking about how to make it better. Shielded transactions support and UX is full of tradeoffs in general.